packages/dev/s2-docs/src/CodePlatter.tsx

```diff
@@ -250,6 +250,27 @@ function getExampleCode(codeRef: RefObject<HTMLDivElement | null>, urls: {[name:
     }
   }
 
+  if (code.includes('export default function')) {
```

Comment:
apologies, didn't mean to push this to your PR, meant to put it in it's own branch off of it. But I noticed that the react aria "examples" such as the emoji picker, were broken. If there were multiple functions, then they were all being renamed to Example and exported as the default from the same module

Reply:
can we deduplicate this code with the below? If we only need to skip the first replace:

```tsx
if (code.includes('export default function')) {
  code = code.replace(/\nfunction ([^(]+)((.|\n)+\n\}\n?)$/, '\nexport default function Example$2');
}
```